templates/vue/src/components/modals/NodeModal/forms/ContentForm/TextForm.vue

```diff
@@ -19,6 +19,11 @@ export default {
       type: Object,
       required: true,
     },
+    // textInput: {
+    //   type: String,
+    //   required: false,
+    //   default: "",
+    // },
```

Comment:
Please do not leave commented out code or if you have to, please leave an explanation of why it's commented out.